elixir/apps/domain/test/domain/relays_test.exs

```diff
@@ -786,18 +786,19 @@ defmodule Domain.RelaysTest do
     end
   end
 
-  describe "generate_username_and_password/1" do
+  describe "generate_username_and_password/3" do
     test "returns username and password", %{account: account} do
```

Comment:
I think this test (or another) would be better if it would assert the actual output against static strings. That way, we would assert that it is pure and does not contain any randomness. You have to pass a fixed `expires_at` of course.

Reply:
Good call